README.md

```diff
@@ -30,9 +30,9 @@ _Note: The version numbers below are only examples, meant to illustrate the rela
 | ------------------------- | -------------------------- |------------------- | ------- |
 | Versions are the same.    | 5.1.2                      | 5.1.2              | 💚 OK      |
 | ES patch number is newer. | 5.1.__2__                  | 5.1.__5__          | ⚠️ Logged warning      |
+| ES patch number is older. | 5.1.__2__                  | 5.1.__0__          | ⚠️ Logged warning      |
```

Comment:
I think we should keep this placement in the original order. I think people will be scanning for conditions and then looking for outcomes, instead of the other way around. In the original order, there's a repeating pattern to add scanning.